kerlescan/inventory_service_interface.py

```diff
@@ -141,6 +141,15 @@ def fetch_systems_with_profiles(system_ids, service_auth_key, logger, counters):
 
     ensure_correct_system_count(system_ids, systems_result)
 
-    return interleave_systems_and_profiles(
+    systems_with_profiles = interleave_systems_and_profiles(
         systems_result, system_profiles_result, system_tags_result
     )
+
+    # remove CentOS Linux systems
```

Comment:
The below code selects and not deletes, as this comment explains.

Reply:
Well, it's Friday... thank you